includes/footer.html

```diff
@@ -60,7 +60,14 @@ <h5 class="white-text fine">coala</h5>
     </div>
     <div class="footer-copyright">
         <div class="container">
-            <span class="left">© 2017 coala</span>
+            <span class="left">
+	        ©
+		<script>
+		    document.getElementById("current-year").innerHTML=new Date().getFullYear();
+		</script>
+		<span id="current-year"></span>
+                coala
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpl9zy7y_2/includes/footer.html
+++ b/tmp/tmpl9zy7y_2/includes/footer.html
@@ -63,7 +63,7 @@
             <span class="left">
 	        ©
 		<script>
-		    document.getElementById("current-year").innerHTML=new Date().getFullYear();
+            document.getElementById("current-year").innerHTML=new Date().getFullYear();
 		</script>
 		<span id="current-year"></span>
                 coala
```